vowpalwabbit/kernel_svm.cc

```diff
@@ -872,8 +872,8 @@ LEARNER::base_learner* kernel_svm_setup(vw &all)
   else
     params.kernel_type = SVM_KER_LIN;
 
-  params.all->reg.weight_mask = (uint64_t)LONG_MAX;
-  params.all->reg.stride_shift = 0;
+  //params.all->weights->mask((uint64_t)LONG_MAX); 
```

Comment:
The mask is calculated using the length and the stride_shift so I don't think there should be a setter for mask.